src/engine/controls/cuecontrol.cpp

```diff
@@ -498,6 +507,10 @@ void CueControl::loadCuesFromTrack() {
             }
             // Add the hotcue to the list of active hotcues
             active_hotcues.insert(hotcue);
+            break;
+        }
+        default:
+            continue;
```

Comment:
we have here a mismatch between use of beak and continue. 
If I get it right, both have the same effect, so we should use one. I would prefer break, since that is the normal switch pattern.

Reply:
ok